raven/src/main/java/org/opencadc/raven/InitDatabaseAction.java

```diff
@@ -0,0 +1,247 @@
+/*
```

Comment:
Why was this whole file copied from minoc? raven is not responsible for creating/updating the database and it's obvious that initStorageSite() is completely wrong at a global inventory site. 

I know we agreed to copy the config handling but that's only a subset of this class and it is in a separate class in minoc because that service does need to init on startup. An InitAction might be a nice way to achieve "read once" config here, but this class does not do that and is not even executed by adding it to web.xml ... so as it is we **might only** have a really confusing way to add a static method for reading config.

Reply:
I misunderstood then. I thought the idea was copy the class, then later move common setup/config code to a util library. Removed the class.